### PR TITLE
[MFTF] Add  new AdminSelectAndDeleteProductsActionGroup

### PR DESCRIPTION
**Pull request by Usik2203**
This PR adds new `AdminSelectAndDeleteProductsActionGroup` 
This action group will select all product and remove them

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@rogyar Could you review this one please ?
Thanks

File: app/code/Magento/Bundle/Test/Mftf/ActionGroup/AdminSelectAndDeleteProductsActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSelectAndDeleteProductsActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Bundle/Test/Mftf/ActionGroup/AdminSelectAndDeleteProductsActionGroup.xml]: I would suggest a little bit different name. Something like `AdminDeleteAllProductsFromGridActionGroup`. 
Also, what do you think about moving this action group to the Catalog module?

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests B2B

File: app/code/Magento/Bundle/Test/Mftf/ActionGroup/AdminSelectAndDeleteProductsActionGroup.xml (hunk 1 above)
Comment (Usik2203) [reply app/code/Magento/Bundle/Test/Mftf/ActionGroup/AdminSelectAndDeleteProductsActionGroup.xml]: **Done**

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-8002](https://jira.corp.magento.com/browse/ENGCOM-8002) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Pull request merged**